### ISSUE TITLE
give loss and val_loss to nan

### ISSUE THREAD

**Issue by Shuaibiqbal** [krishnaik06/Stock-MArket-Forecasting#1]
i am trying to prediction on household power consumption dataset
your given model give loss and val_loss to nan when i run this code ..please elaborate it
my dataset is 200k
when i select only first 1000 data for just checking then it give some values on loss and val_loss but when i increase dataset it give loss and val_loss to nan ... whats the reason

please response ASP

**Comment by KaranRawat1998** [comment]
check if there are any nan values in the data set thats why it is giving nan in loss and val loss
